sdk/webpubsub/Azure.Messaging.WebPubSub/src/NS2Bridge.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Text;
+
+namespace Azure.Core
+{
+    // APIs not avaliable in NetStandard 2.0
+    internal static class NS2Bridge
+    {
+        public delegate void SpanAction<T, TArg>(Span<T> buffer, TArg state);
+        public static string CreateString<TState>(int length, TState state, SpanAction<char, TState> action)
+        {
+            var result = new string((char)0, length);
+            unsafe
+            {
+                fixed (char* chars = result)
+                {
+                    var charBuffer = new Span<char>(chars, result.Length);
+                    action(charBuffer, state);
+                }
+            }
+            return result;
+        }
+
+        public static void Append(this StringBuilder sb, ReadOnlySpan<char> value)
```

Comment:
Yeah, I had a WriteTo(StringBulder) overload. But I removed it. If we ever make the JwtBulder public, it would be good to have these, but I will add them at that point.